src/main/java/io/druid/data/input/MapBasedRow.java

```diff
@@ -164,11 +151,8 @@ public boolean equals(Object o)
     if (!event.equals(that.event)) {
       return false;
     }
-    if (!timestamp.equals(that.timestamp)) {
-      return false;
-    }
+    return timestamp.equals(that.timestamp);
```

Comment:
@okrische that's funny :) I guess IntelliJ optimized its generator then, feel free to leave it.